yarn.lock

```diff
@@ -17437,6 +17498,11 @@ typescript@^4.1.3:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-4.1.3.tgz#519d582bd94cba0cf8934c7d8e8467e473f53bb7"
   integrity sha512-B3ZIOf1IKeH2ixgHhj6la6xdwR9QrLC5d1VKeCSY4tvkqhF2eqd9O7txNlS0PO3GrBAFIdr3L1ndNwteUbZLYg==
 
+typescript@^4.2.3:
```

Comment:
try run deduplicate for this repository, I see that you have added new version of typescript but previous version was not removed

Reply:
Add deduplicate script in root package.json